lib/registration.pm

```diff
@@ -117,7 +117,7 @@ sub fill_in_registration_data {
             record_soft_failure('bsc#1056413');
             # Add expected modules to select them manually, as not preselected
             my %addons = (
-                sles => 'base,script,desktop,serverapp',
+                sles => 'base,script,desktop,serverapp,we',
```

Comment:
It's wrong to add the module here because this is the workaround to select modules which should be preselected to form the standard SLES installation, see bugref bsc#1056413 above

Reply:
This is OK. We can add SCC_ADDON=base,we as parameter to the new test scanario that is to be created.